appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/interactions/core/ui/state/MutableUiStateSpecs.kt

```diff
@@ -0,0 +1,12 @@
+package com.bumble.appyx.interactions.core.ui.state
+
+@Target(AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.SOURCE)
+@Suppress("Unused")
+annotation class MutableUiStateSpecs(
+    val playMode: PlayMode = PlayMode.ConcurrentMode
+) {
+    enum class PlayMode {
+        SequentialMode, ConcurrentMode
+    }
```

Comment:
Maybe I'd rename to:

```suggestion
    enum class AnimationMode {
        SEQUENTIAL, CONCURRENT
    }
```